package.json

```diff
@@ -28,9 +28,9 @@
         "preversion": "npm run build && npm run lint && npm run test && npm run check-extraneous && node scripts/preversion.js",
         "vscode:prepublish": "npm run build",
         "copy-schema": "cpx ./node_modules/brighterscript/bsconfig.schema.json ./dist -L",
-        "build": "npm run copy-schema && tsc -p ./",
+        "build": "npm run copy-schema && tsc -p ./ && npm run svelte-build",
         "lint": "tslint \"./src/**/*.ts\"",
-        "watch": "npm run copy-schema && tsc -w -p ./",
+        "watch": "npm run copy-schema && tsc -w -p ./ && npm run svelte-build",
```

Comment:
Will this run? I think the tsc will prevent the build until it's done watching

Reply:
Sorry that was some testing thing will remove it

Reply:
it did not throw any error though